### DISCUSSION TITLE
Island level

### DISCUSSION THREAD

**Discussion by AlexH1705** [BG-Software-LLC/SuperiorSkyblock2#1774] [Pending Feature Requests]
i am trying to modify the island level in the config with the worth formula...but if i want to do worth/10 the level is negative (i have minecraft 1.16.5 with NashornEngine installed)
![image](https://github.com/BG-Software-LLC/SuperiorSkyblock2/assets/86449461/11b1d607-2a0b-4157-bfc1-f65376a54fef)
![image](https://github.com/BG-Software-LLC/SuperiorSkyblock2/assets/86449461/4c46bc86-cc30-4076-a656-431781bc4977)

**Comment by Tuogptuan** [comment]
So disable negative-level, iirc?

**Reply by AlexH1705** [reply]
Wdym...worth/10 is 246,9 not -1383,5 💀 what are you talking about

**Comment by Tuogptuan** [comment]
Try syncbonus and pls move to issues page.
